services/venice-router/src/main/java/com/linkedin/venice/router/MetaDataHandler.java

```diff
@@ -316,6 +316,14 @@ private void handleValueSchemaLookup(ChannelHandlerContext ctx, VenicePathParser
       int index = 0;
       for (SchemaEntry entry: valueSchemaEntries) {
         int schemaId = entry.getId();
+        if (schemaId < 1) {
+          LOGGER.warn(
+              "Got an invalid schema id ({}) for store {} in handleValueSchemaLookup; will not include this in the {}.",
+              entry.getId(),
+              storeName,
+              MultiSchemaIdResponse.class.getSimpleName());
```

Comment:
In this case, the response is of type `MultiSchemaResponse`. Instead of getting the class statically, it might be worth changing this to `responseObject.getClass().getSimpleName()` instead.

Reply:
Sure